cmd/karavictl/cmd/storage_create_test.go

```diff
@@ -335,3 +339,107 @@ func Test_readPassword(t *testing.T) {
 		}
 	})
 }
+
+func TestStorageCreateHandler(t *testing.T) {
+	afterFn := func() {
+		CreateHTTPClient = createHTTPClient
+		JSONOutput = jsonOutput
+		osExit = os.Exit
+	}
+
+	t.Run("it requests creation of a storage", func(t *testing.T) {
+		defer afterFn()
+		var gotCalled = true
+		CreateHTTPClient = func(addr string, insecure bool) (api.Client, error) {
+			return &mocks.FakeClient{
+				PostFn: func(ctx context.Context, path string, headers map[string]string, query url.Values, body, resp interface{}) error {
+					gotCalled = true
+					return nil
+				},
+			}, nil
+		}
+		JSONOutput = func(w io.Writer, _ interface{}) error {
+			return nil
+		}
+		osExit = func(code int) {
+		}
+		var gotOutput bytes.Buffer
+
+		cmd := NewRootCmd()
+		cmd.SetOutput(&gotOutput)
+		cmd.SetArgs([]string{"storage", "create", "--addr", "https://storage-service.com", "--endpoint", "https://0.0.0.0:443", "--system-id", "testing123", "--type", "powerflex", "--user", "admin", "--password", "password", "--insecure", "--array-insecure"})
+		cmd.Execute()
+
+		if !gotCalled {
+			t.Error("expected DeleteTenant to be called, but it wasn't")
```

Comment:
Is it DeleteTenant to be called? Also the gotCalled was set to true anyways from the beginning?

Reply:
fixed, also noticed this on other karavictl test and updated the error logs